broker/src/main/java/io/camunda/zeebe/broker/system/SystemContext.java

```diff
@@ -40,6 +40,12 @@ public final class SystemContext {
       "Node id %s needs to be non negative and smaller then cluster size %s.";
   private static final String REPLICATION_FACTOR_ERROR_MSG =
       "Replication factor %s needs to be larger then zero and not larger then cluster size %s.";
+  private static final String REPLICATION_FACTOR_WARN_MSG =
+      "Expected to have odd replication factor, but was even ({}). Even replication factor has no benefit over "
+          + "the next odd value. Quorum is calculated as:"
```

Comment:
```suggestion
          + "the previous odd value. Quorum is calculated as:"
```

Reply:
Actually I meant next, because the quorum value will be the same.

Reply:
I understand but I think it's still correct and important to say: "replication factor 2 is not better than 1 and it's worse than 3, in most cases you should use 1 or 3"

If we only call out that 2 is not better than 3 ("Even replication factor has no benefit over the next odd value"), it sounds like it is just as good as 3 where it actually is worse.

Reply:
Thanks @oleschoenburg for the feedback I adjusted the warning.